### DISCUSSION TITLE
Replay M6 : Index error

### DISCUSSION THREAD

**Discussion by titi-17** [Catch-up-TV-and-More/plugin.video.catchuptvandmore#1058] [General]
Bonjour,
Je ne peux plus lire les replay de M6
Je sélectionne la vidéo et lorsque je veux la lancer, j'ai le message "Indexerror"
J'ai essayé d'envoyer le rapport d'erreur pour le partager, mais impossible.
Catch up est installer sur une clé MI android.
Pouvez vous m'aider?
Merci d'avance.

**Comment by darodi** [comment]
M6 group 
Doublon, faire une mise à jour vers la version beta 17

- #1042
- #1039
- #1044 
- #1046 
- #1049 
- #1057 
- #1055

Il est préférable de faire un ticket dans issue au lieu de créer une discussion
